docs/backup_and_restore.rst

```diff
@@ -160,7 +160,7 @@ backup:
 .. code:: sh
 
    cd install_files/ansible-base
-   ansible-playbook -i inventory -u <SSH username> -K -t backup securedrop-prod.yml -e restore_file="<your backup archive filename>"
+   ansible-playbook -u <SSH username> -K -t backup securedrop-prod.yml -e restore_file="<your backup archive filename>"
```

Comment:
Let's move to another issue. It's a great idea to standardize on that var, but we'd almost surely break backwards compatibility to do so. Will need to think about the smartest way to handle the YAML parsing.